test/test_literate.litcoffee

```diff
@@ -34,6 +32,22 @@ The 3rd parameter here indicates that the incoming source is literate.
 This intentionally includes trailing whitespace in code so it also verifies
 that the way `Markdown` spaces are stripped are not also stripping code.
 
-                assert.equal(errors.length, 2)
+                assert.equal(errors.length, 1)
+
+        'Tab indented markdown' :
```

Comment:
nit: can you remove the space between the single quote and ':'? Probably should turn this on in the linter config.